modules/manage/partials/authentication.adoc

```diff
@@ -0,0 +1,1661 @@
+----
+
+If TLS is enabled, specify the HTTPS protocol and pass the path to the `ca.crt` file:
+
+[source,bash]
+----
+kubectl exec <pod-name> --namespace <namespace> -- curl https://redpanda-0.redpanda.redpanda.svc.cluster.local:8082/topics --cacert /etc/tls/certs/default/ca.crt -u <username>:<password> -sS
+----
+
+NOTE: If the broker's certificate is signed by a well-known, trusted CA, and you're confident about the integrity of your system's CA trust store, you don't need the `--cacert` flag.
+endif::[]
+ifndef::env-kubernetes[]
+
+[source,bash]
+----
+curl http://localhost:8082/topics -u <username>:<password> -sS
+----
+
+If TLS is enabled, specify the HTTPS protocol and pass the path to the `ca.crt` file:
+
+[source,bash]
+----
+curl https://localhost:8082/topics --cacert <path-to-ca>/ca.crt -u <username>:<password> -sS
+----
+
+NOTE: If the broker's certificate is signed by a well-known, trusted CA, and you're confident about the integrity of your system's CA trust store, you don't need the `--cacert` flag.
+endif::[]
+
+For all available endpoints, see xref:api:ROOT:pandaproxy-rest.adoc[].
+
+==== Connect to the Schema Registry API
+
+ifdef::env-kubernetes[By default, the Redpanda Helm chart configures an internal and external listener for the Schema Registry API.]
+
+To access the internal listener:
+
+ifdef::env-kubernetes[]
+[source,bash]
+----
+kubectl exec <pod-name> --namespace <namespace> -- curl http://redpanda-0.redpanda.redpanda.svc.cluster.local:8081/subjects -u <username>:<password> -sS
+----
+
+If TLS is enabled, specify the HTTPS protocol and pass the path to the `ca.crt` file:
+
+[source,bash]
+----
+kubectl exec <pod-name> --namespace <namespace> -- curl https://redpanda-0.redpanda.redpanda.svc.cluster.local:8081/subjects --cacert /etc/tls/certs/default/ca.crt -u <username>:<password> -sS
+----
+
+NOTE: If the broker's certificate is signed by a well-known, trusted CA, and you're confident about the integrity of your system's CA trust store, you don't need the `--cacert` flag.
+endif::[]
+ifndef::env-kubernetes[]
+
+[source,bash]
+----
+curl http://localhost:8081/subjects -u <username>:<password> -sS
+----
+
+If TLS is enabled, specify the HTTPS protocol and pass the path to the `ca.crt` file:
+
+[source,bash]
+----
+curl https://localhost:8081/subjects --cacert <path-to-ca>/ca.crt -u <username>:<password> -sS
+----
+
+NOTE: If the broker's certificate is signed by a well-known, trusted CA, and you're confident about the integrity of your system's CA trust store, you don't need the `--cacert` flag.
+endif::[]
+
+For all available endpoints, see xref:api:ROOT:pandaproxy-schema-registry.adoc[].
+
+[[oidc-http]]
+=== OIDC
+
+If you've configured the <<oidc, SASL/OAUTHBEARER>> mechanism for the Kafka API, you can configure the HTTP APIs to authenticate users with the OIDC bearer token. By using OIDC for all HTTP APIs, you centralize credentials and provide a password-free,
```

Comment:
You can configure OIDC for the HTTP APIs without having configured it for Kafka API. (the `oidc_*` cluster configs apply here, too.)